src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
would be nice to add a general class for similar elements instead of styling with a tag